### PR TITLE
QAE-257 - E2E test case for off canvas flyout general settings for tablet and mobile mode

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Added test case for off canvas general settings for mobile and tablet mode.

### Screenshots

Alignment : https://share.getcloudapp.com/X6ubw4Zw

### How has this been tested?

npm run test:e2e:interactive tests/e2e/specs/customizer/header-builder/mobile-header/off-canvas-header/flyout-general.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/header-builder/mobile-header/off-canvas-header/flyout-general.test.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
+import { createURL, createNewPost, publishPost } from '@wordpress/e2e-test-utils';
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/mobile-header/off-canvas-header/flyout-general.test.js:1]: publish-post util needs to update

File: tests/e2e/specs/customizer/header-builder/mobile-header/off-canvas-header/flyout-general.test.js [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+import { setCustomize } from '../../../../../utils/customize';
+import { setBrowserViewport } from '../../../../../utils/set-browser-viewport';
+describe( 'off canvas flyout header type settings in the customizer', () => {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/mobile-header/off-canvas-header/flyout-general.test.js:4]: first letter should be capital

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/header-builder/mobile-header/off-canvas-header/flyout-general.test.js]: Done

File: tests/e2e/specs/customizer/header-builder/mobile-header/off-canvas-header/flyout-general.test.js (hunk 1 above)
Comment (SimranChougule1) [reply tests/e2e/specs/customizer/header-builder/mobile-header/off-canvas-header/flyout-general.test.js]: Done

**Review by bsfsnehal — APPROVED** [review]

**Review by GauravKhupse — APPROVED** [review]

**Pull request merged**